### PR TITLE
Add separate page for the java support

### PR DESCRIPTION
**Pull request by v-thakkar**
Let's have individual pages for the different languages so that we can elaborate more on an individual
langauge support in parca-agent.

This commit adds one such page for the java support. In the future we will have seperate pages for other langauges so that users don't have figure it out by looking into demo Dockerfiles.

### CODE REVIEW

**Review by kakkoyun — APPROVED** [review]

File: sidebars.js [hunk 1]
```diff
@@ -68,7 +68,11 @@ module.exports = {
         "parca-agent",
         "parca-agent-design",
         "parca-agent-labelling",
-        "parca-agent-language-support",
+        {
+          type: "category",
+          label: "Language Support",
```
Comment (metalmatze) [inline sidebars.js:73]:
```suggestion
          label: "Languages",
```
Personally, I'd prefer a shorter menu name here.
Great idea with the languages!

Comment (v-thakkar) [reply sidebars.js:73]: I agree with respect to having shorter menu names whenever possible. Although in this case, I think just 'Languages' can be a bit confusing and generic in the context of parca-agent.

File: docs/java-support.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Java Support
+
+In order for Parca Agent to be able to make sense of the just-in-time compiled code by the Java VM, the Java process needs to comply to the [linux kernel perf jit-interface](https://github.com/torvalds/linux/blob/master/tools/perf/Documentation/jit-interface.txt). To do this, currently users need to start their java process with two flags -XX:+PreserveFramePointer and -agentpath:/your/path/to/libperfmap.so where the libperfmap.so agent can be downloaded [here](https://github.com/parca-dev/perf-map-agent/releases/tag/v0.0.1).
```
Comment (brancz) [inline docs/java-support.md:3]: For this to render the way we want we need to wrap the flags in backticks.

File: docs/parca-agent-language-support.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ All JIT compiled languages with frame pointer, and [Perf map](https://github.com
 * [.NET](https://github.com/parca-dev/parca-demo/tree/main/dotnet)
 * [Deno](https://github.com/parca-dev/parca-demo/tree/main/deno)
 * [Erlang](https://github.com/parca-dev/parca-demo/tree/main/erlang)
-* [Java](https://github.com/parca-dev/parca-demo/tree/main/java)
+* [Java](docs/java-support)
```
Comment (brancz) [inline docs/parca-agent-language-support.md:22]: This is what's causing the build to fail (docusaurus correctly detected that this link would be broken as is as it's relative to what is already the `/docs` path. I think this either needs to be just `java-support` or `/docs/java-support`.

Comment (v-thakkar) [reply docs/parca-agent-language-support.md]: Oops, my bad! I was overconfident on my Markdown skills so didn't test locally in the morning. :see_no_evil:

**Review by brancz — APPROVED** [review]

**Pull request merged**